packages/core/README.md

```diff
@@ -44,3 +44,25 @@ yarn add @types/styled-components @types/xstyled__styled-components
 ```
 
 You can find a list of all available components at our [Storybook](https://react95.github.io/React95/)
+
+### SSR
```

Comment:
Not necessarily SSR, right? This is a Next.js example.

Reply:
Done!